WGS84ReferenceFrame.h

```diff
@@ -0,0 +1,90 @@
+//####################################################################
+//
+//    FILENAME:          WGS84ReferenceFrame.h
+//
+//    CLASSIFICATION:    Unclassified
+//
+//    DESCRIPTION:
+//
+//    Header for WGS84 reference frame realization and epoch.
+//
+//    LIMITATIONS:       None
+//
+//
+//    SOFTWARE HISTORY:
+//     Date          Author    Comment
+//     -----------   ------    -------
+//     31-December-2021   Eugene Rose CCB Change
+//
+//    NOTES: 
+//    As of writing this, these are the WGS84 realizations to date.
+//    WGS84 ORIGINAL 1984.0
+//    G730           1994.0
+//    G873           1997.0
+//    G1150          2001.0
+//    G1674          2005.0
+//    G1762          2005.0
+//
+//###################################################################
+
+#ifndef __CSM_WGS84REFERENCEFRAME_H
+#define __CSM_WGS84REFERENCEFRAME_H
+
+#include "csm.h"
+#include <string>
+
+#define WGS84_ORIGINAL "WGS84 ORIGINAL"
+#define WGS84_G730 "G730"
+#define WGS84_G873 "G873"
+#define WGS84_G1150 "G1150"
+#define WGS84_G1674 "G1674"
+#define WGS84_G1762 "G1762"
+
+namespace csm
+{
+
+	class CSM_EXPORT_API WGS84ReferenceFrame
+	{
+	public:
+		
+		struct CSM_EXPORT_API realizationDefinition {

```

Comment:
Other types are in capitalized camel case, like `RasterGM` or `Plugin`.  This type should probably be `RealizationDefinition`, especially since it is a public type.